web-app/.vscode/settings.json

```diff
@@ -14,10 +12,6 @@
   "files.associations": {
     "*.css": "tailwindcss"
   },
-  "prettier.tabWidth": 2,
-  "prettier.semi": true,
-  "prettier.singleQuote": false,
-  "prettier.trailingComma": "all",
```

Comment:
> I think without this change, prettier formatted it wrong for me. Is it necessary to remove it?

By @schaier-io

Reply:
We have two sources of truth. Prettier has a configuration `prettier.config.mjs`.
I would prefer to have one source of truth.

Could it be a configuration issue in your IDE?

Reply:
ok lets try it for now, if I does not work at all, I will revert it